### DISCUSSION TITLE
Issue with Enum Mapping in Prisma Client

### DISCUSSION THREAD

**Discussion by nilaypsoni** [prisma/prisma#22486] [Q&A]
### Question

Hello Prisma Community,

I hope this message finds you well. I am currently facing an issue with enum mapping in Prisma Client, and I would appreciate any insights or guidance you might have.

Problem:
When using Prisma Client-generated methods to fetch data, the enum field (time_period) returns the enum values ('S_1920_1930') instead of the mapped values ('1920 - 1930').

Details:

Prisma schema includes enum mappings using @map directive.
Prisma Client has been regenerated after making changes to the schema.
I have verified that the raw query returns the expected mapped values.
Prisma Client Version:
I am using Prisma Client version 5.7.1.
prisma 5.7.1
@prisma/client  : 5.7.1

### How to reproduce (optional)

// Prisma schema with enum mappings
enum user_time_period_enum {
  S_1920_1930 @map("1920 - 1930")
  // ... other enum values
}

const users = await prisma.user.findMany();
console.log('Fetched Users:', users);

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hello @nilaypsoni 👋 

Thank you for raising this request.

We currently have an open [issue](https://github.com/prisma/prisma/issues/8446) that describes the exact issue you are facing where we are not returning the mapped value of the Enum. As a workaround, you can use the raw query like you have described. You can also add your usecase to the open issue and leave a 👍 on it to help our engineers prioritize it.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!

**Comment by justin-rwazi** [comment]
it've been 2 years and I still got this issue
can't believe that
